includes/api/v2/class-wc-rest-product-categories-v2-controller.php

```diff
@@ -62,11 +62,14 @@ public function prepare_item_for_response( $item, $request ) {
 				'date_created_gmt'  => wc_rest_prepare_date_response( $attachment->post_date_gmt ),
 				'date_modified'     => wc_rest_prepare_date_response( $attachment->post_modified ),
 				'date_modified_gmt' => wc_rest_prepare_date_response( $attachment->post_modified_gmt ),
-				'src'               => wp_get_attachment_url( $image_id ),
+				'src'               => wc_get_term_src_url( $item, $image_id, false, false ),
 				'title'             => get_the_title( $attachment ),
 				'alt'               => get_post_meta( $image_id, '_wp_attachment_image_alt', true ),
 			);
 		}
+		else{
```

Comment:
This does not follow coding standards.

Reply:
Mike, sorry for bothering, can you modify it as it needs to be? Because I am confused..

Reply:
You pointed me to the "braces" part in WP Coding Standards, but i don't see any braces related to these parts.. ( 
Probably it's not long when I started GitHub and I have problem to understand which parts you point me..